Islandora-Devops/migrate_7x_claw
# Add most known mime types and alphabetize them

**GitHub Issue**: N/A

# What does this Pull Request do?
Includes new mime type to allow for the automatic assignment of unexpected formats.

# What's new?
Adds several known mime types and alphabetize them.

# How should this be tested?
Add a file type that matches one of the new mime types into an Islandora 7 size and see if the migration recognizes it when previously it would have failed. 

# Additional Notes:
N/A

# Interested parties
@Islandora-Devops/committers
